dMasterServer/MasterServer.cpp

```diff
@@ -187,15 +188,20 @@ int main(int argc, char** argv) {
 		std::cout << "Enter a username: ";
 		std::cin >> username;
 
+		const auto checkIsAdmin = []() {
+			std::string admin;
+			std::cout << "What level of privilege should this account have? Please enter a number between 0 and 9 inclusive. No entry will default to 0." << std::endl;
```

Comment:
```suggestion
			std::cout << "What level of privilege should this account have? Please enter a number between 0 and 9 inclusive. No entry will default to 0." << std::endl;
```
```suggestion
			std::cout << "What level of privilege should this account have? Please enter a number between 0 (Player) and 9 (Admin) inclusive. No entry will default to 0." << std::endl;
```

Reply:
Fwiw, enum calls them `Civilian` / "Normal Player" and `Operator`  "can shut down server"

Reply:
I had minecraft on the brain 🙈